### DISCUSSION TITLE
Oculus / VR

### DISCUSSION THREAD

**Discussion by ronvanherk** [rustdesk/rustdesk#3047] [Ideas]
Hi,

Is there anyone working on a port to the Meta Oculus - or something similar - yet?

Best regards,
Ron

**Comment by jaiotu** [comment]
I have sideloaded the rustdesk apk onto the Meta Quest 2 and so far it works fabulously. Porting it should be a breeze.

**Reply by jurassicjordan** [reply]
youve had no problems with mouse input?
